src/main/java/org/folio/rest/service/action/RefundActionService.java

```diff
@@ -164,6 +164,7 @@ private static void updateAccountInMemory(Account account, Feefineaction feeFine
     account.setRemaining(feeFineAction.getBalance());
     account.getPaymentStatus().setName(fromValue(feeFineAction.getTypeAction()));
     account.getStatus().setName(OPEN.getValue());
+    account.setDateClosed(null);
```

Comment:
Can `dateClose` change here from some non-null value to null? If it can, is there a test for this scenario?

Reply:
Yes, there is already a test that goes through this code.

I just added the "Account" status validation, where it causes the close date to change to null ([commit](https://github.com/folio-org/mod-feesfines/pull/213/commits/986d4122b1b215023b91ffba63cddf683ba19454))